PixivUtil2.py

```diff
@@ -1272,10 +1276,10 @@ def setup_option_parser():
 10 - Download by Tag and Member Id
 11 - Download images from Member Bookmark
 12 - Download images by Group Id
-f1 - Download from supported artists (FANBOX)
+f1 - Download from supporting artists (FANBOX)
 f2 - Download by artist/creator id (FANBOX)
 f3 - Download by post id (FANBOX)
-f4 - Download from followed artists (FANBOX)
```

Comment:
grammar wise should be in passive form, right? not sure if due to british english/american english grammar difference...

Reply:
i don't know anymore... i am confused by it now.
either way seems a little weird
if you think it should be passive form, please change it for me..
thanks

Reply:
since the request is closed already